### DISCUSSION TITLE
Accidentally delete the initial renovate/configure will cause renovate to not access it again

### DISCUSSION THREAD

**Discussion by hunkeelin** [renovatebot/renovate#22378] [Request Help]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

Example: https://github.com/hunkeelin/pki/pull/1

I am unable to reinitiate the PR from the dashboard again. Any tips to get it setup if someone accidentally delete the first configure PR and its branch?

### What you tried so far.

Nothing in the dashboard indicate i can reinitiate the initial PR. 

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment]
This could work:
- Rename the PR (e.g. to "Configure Renovate - old")
- Uninstall, then reinstall, the repo

**Reply by hunkeelin** [reply]
How you uninstall and reinstall? Theres no such button in the dashboard

**Reply by rarkins** [reply]
https://github.com/apps/renovate
